[IMP] Purchase: Update RFQ with dependency warnings and new images (#12682)
odoo/documentation: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

-------

To auto-populate product information and prices on an |RFQ|, configure products by going to
:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a new
one by selecting :guilabel:`New`. Doing so opens the product form, where sales and purchasing data
can be configured.

To configure purchasable products, tick the :guilabel:`Purchase` checkbox, under the product name.
Next, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.

.. important::
   The :guilabel:`Inventory` tab and routes are only visible if using the :doc:`Inventory

>    app<../../inventory>`.

Reviewer: Required change: we need a space between the link text and link (:

Optional suggestion:

Another important field to configure on the product form is the product type and track inventory. It's covered in this [Product type doc](https://runbot114.odoo.com/runbot/static/build/77396661-18-0/logs/build/html/applications/inventory_and_mrp/inventory/product_management/configure/type.html), so maybe we can include a seealso, that contains info about those fields?

```suggestion
   app <../../inventory>`.
   
.. seealso::
   :doc:`Configure product types and track quantities <TODO: link to doc>`
```

.. image:: rfq/product-vendor-pricelist-config.png

>       :alt: Required configuration for purchasable products.

Reviewer: ```suggestion
   :alt: Required configuration for purchasable products.
```

.. _purchase/manage_deals/vendor-pricelist:

Vendor pricelist
----------------

In the :guilabel:`Purchase` tab of the product form, click :guilabel:`Add a line` to input the
vendor and their price, to have this information auto-populate on an |RFQ| each time the product is
listed.

.. seealso::
   :doc:`../products/pricelist`

Default columns include :guilabel:`Quantity`, :guilabel:`Unit Price`, and :guilabel:`Delivery Lead
Time`, but other columns like, :guilabel:`Vendor Product Code` or :guilabel:`Discount (%)`, can also
be enabled.

To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(additional options)`
icon on the right side of the header row to reveal a drop-down menu of additional columns that can
be added (or removed) from the :guilabel:`Purchase` tab.

.. note::

> 

Reviewer: Delete extra line (:
```suggestion
```

-------

To auto-populate product information and prices on an |RFQ|, configure products by going to
:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a new
one by selecting :guilabel:`New`. Doing so opens the product form, where sales and purchasing data
can be configured.

To configure purchasable products, tick the :guilabel:`Purchase` checkbox, under the product name.
Next, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.

.. important::
   The :guilabel:`Inventory` tab and routes are only visible if using the :doc:`Inventory app
   <../../inventory>`.

.. seealso::
   :doc:`Configure product types and track quantities <../../../inventory_and_mrp/inventory/product_management/configure>`

.. image:: rfq/product-vendor-pricelist-config.png
   :alt: Required configuration for purchasable products.

.. _purchase/manage_deals/vendor-pricelist:

Vendor pricelist
----------------

In the :guilabel:`Purchase` tab of the product form, click :guilabel:`Add a line` to input the
vendor and their price, to have this information auto-populate on an |RFQ| each time the product is

> listed.

Reviewer 2: Suggest to pluralize "vendor" to "vendors" and then explain how the order in the list matters (the vendor at the top of the list is preferred across other operations like reordering rules)

.. seealso::
   :doc:`../products/pricelist`

Default columns include :guilabel:`Quantity`, :guilabel:`Unit Price`, and :guilabel:`Delivery Lead
Time`, but other columns like, :guilabel:`Vendor Product Code` or :guilabel:`Discount (%)`, can also
be enabled.

> To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(additional options)`
> icon on the right side of the header row to reveal a drop-down menu of additional columns that can
> be added (or removed) from the :guilabel:`Purchase` tab.

Reviewer 2: Essentials content, can remove
It's better to more deeply explain the what default columns mean and how to fill them in correctly.
```suggestion
```

.. note::

>    Alternatively, prices and delivery lead times for existing products can be added by going to

Reviewer 2: The explicit "bulk" part here is a useful tip, I don't see the need to remove it?

- :guilabel:`Late`: |RFQs| or |POs| where the :guilabel:`Order Deadline` has passed.

.. image:: rfq/rfq-dashboard.png

>       :alt: RFQ dashboard with orders and order statuses.

Reviewer: This is SUCH an annoying and nitpicky thing, but we avoid tabs and use THREE spaces (yes i counted :sob:) to align this tag with the "i" in image :sob: 

```suggestion
   :alt: RFQ dashboard with orders and order statuses.
```

Reviewer: For images, I rely on the image snippet (:

![image](https://github.com/user-attachments/assets/8308b484-8eaa-468c-adc7-f6e6a07a1249)
![image](https://github.com/user-attachments/assets/8c5eded1-d0dd-4693-951c-b6490d7126c0)

Author: Is there something else that needs to be changed with the alt text, or is that just a tip to make my life easier in general?

- :guilabel:`Late`: |RFQs| or |POs| where the :guilabel:`Order Deadline` has passed.

.. image:: rfq/rfq-dashboard.png
   :alt: RFQ dashboard with orders and order statuses.

In addition to view options, the :guilabel:`Requests for Quotation` dashboard provides

> :guilabel:`Filters` and :guilabel:`Group By` options, accessible via the search bar drop-down menu.

Reviewer 2: essentials content can remove

vendor. This comes in handy once products are received, and the |PO| needs to be matched to the
delivery order.

> With the :doc:`Purchase Agreements feature <blanket_orders>` activated, the :guilabel:`Blanket

Reviewer 2: I like that we're talking about blanket orders and dropshipping, but at this location in the content, it seems a bit premature since we're simply just trying to create an RFQ? We don't need any of these features enabled in order to create an RFQ...

   After the :guilabel:`Order Deadline` is exceeded, the |RFQ| is marked as late, but the products
   can still be ordered.

:guilabel:`Expected Arrival` is automatically calculated based on the :guilabel:`Order Deadline` and
vendor lead time. Tick the checkbox for :guilabel:`Ask confirmation` to ask the vendor to confirm

> the shipping date by email.

Reviewer: THANK YOU for fixing that inaccuracy about how "Ask confirmation" works

With the :doc:`Storage Locations

> feature<../../inventory/warehouses_storage/inventory_management/use_locations>` activated, the

Reviewer: Adding a space (:
```suggestion
feature <../../inventory/warehouses_storage/inventory_management/use_locations>` activated, the
```

> :guilabel:`Deliver to` field appears, with options for the order shipment.

Reviewer: ```suggestion
:guilabel:`Deliver to` field appears, which specifies which warehouse operation (set in the **Inventory** app) is used to receive the shipment.
```

   :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
   the :guilabel:`Product` name.

Send the |RFQ|
--------------

Clicking :guilabel:`Send by Email` reveals a :guilabel:`Compose Email` pop-up window, with a
:guilabel:`Purchase: Request for Quotation` template loaded, ready to send to the vendor's email
address (configured in the **Contacts** app).

.. TODO: See also for Contacts. Or just say to do it through the vendor form?

> 

Reviewer: Ooooh I found a fellow writer who marks up writing with todos! (: Very dev-y of you :wink: 

I use this vscode extension to highlight my todos so they don't slip through the cracks! Feel free to use it too, if it's helpful to u!

Name: TODO Highlight
Id: wayou.vscode-todo-highlight
Description: highlight TODOs, FIXMEs, and any keywords, annotations...
Version: 1.0.5
Publisher: Wayou Liu
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

Reviewer: Also to answer the question in your TODO:

I think a seealso to the contacts app is good enough (: Putting it at this spot might break up this small section, through, so maybe we can put the block at the end of the section, above "Confirm Order ---------" ?

Author: I think I used the same one, thank you!
